docs/fundamentals/read-operations.txt

```diff
@@ -175,6 +176,124 @@ Use the following syntax to run a find operation that matches all documents:
+whitespace characters. A phrase is a sequence of terms with any number
+of whitespace characters.
+
+.. note::
+
+   Before you can perform a text search, you must create a :manual:`text
+   index </core/indexes/index-types/index-text/>` on
+   the text-valued field. To learn more about creating
+   indexes, see the :ref:`laravel-eloquent-indexes` section of the
+   Schema Builder guide.
+
+After building your query with the ``where()`` method, chain the ``get()``
+method to retrieve the query results.
+
+This example calls the ``where()`` method on the ``Movie`` Eloquent model to
+retrieve documents in which the ``plot`` field contains the phrase
+``"love story"``:
+
+.. tabs::
+
+   .. tab:: Query Syntax
+      :tabid: query-syntax
+
+      Use the following syntax to specify the query:
+
+      .. literalinclude:: /includes/fundamentals/read-operations/ReadOperationsTest.php
+         :language: php
+         :dedent:
+         :start-after: start-text
+         :end-before: end-text
+
+   .. tab:: Controller Method
+      :tabid: controller
+
+      To see the query results in the ``browse_movies`` view, edit the ``show()`` function
+      in the ``MovieController.php`` file to resemble the following code:
+
+      .. io-code-block::
+         :copyable: true
+
+         .. input::
+            :language: php
+
+            class MovieController
+            {
+                public function show()
+                {
+                     $movies = Movie::where('$text', ['$search' => "\"" . 'love story' . "\""])
+                         ->get();
+
+                     return view('browse_movies', [
+                         'movies' => $movies
+                     ]);
+                }
+            }
+
+         .. output::
+            :language: none
+            :visible: false
+
+            Title: Cafè de Flore
+            Year: 2011
+            Runtime: 120
+            IMDB Rating: 7.4
+            IMDB Votes: 9663
+            Plot: A love story between a man and woman ...
+            
+            Title: Paheli
+            Year: 2005
+            Runtime: 140
+            IMDB Rating: 6.7
+            IMDB Votes: 8909
+            Plot: A folk tale - supernatural love story about a ghost ...
+            
+            Title: Por un puèado de besos
+            Year: 2014
+            Runtime: 98
+            IMDB Rating: 6.1
+            IMDB Votes: 223
+            Plot: A girl. A boy. A love story ...
+
+            ...
+            
+.. note::
+
+   To specify a phrase as the text search criteria, you must include it
+   with escaped quotes in the query filter.
+
+A text search assigns a numerical text score to indicate how closely
+each result matches the string in your query filter. You can sort the
+results by relevance by using the ``orderBy()`` method to sort on the
+``textScore`` metadata:
+
+.. literalinclude:: /includes/fundamentals/read-operations/ReadOperationsTest.php
```

Comment:
Suggestion:

I think it could be good to link to a resource that explains $meta and the output. From a quick search, I wasn't able to find a direct explanation, but maybe this [link](https://www.mongodb.com/docs/upcoming/core/indexes/index-types/index-text/control-text-search-results/#assign-weights-to-text-search-results) can help.